### DISCUSSION TITLE
fltk How to close a window via api, is it win.hide?

### DISCUSSION THREAD

**Discussion by Lydanne** [fltk-rs/fltk-rs#1240] [Q&A] [answered]
fltk How to close a window via api, is it win.hide?

**Comment by MoAlyousef** [comment, marked as answer]
Yes. It would also end the fltk event loop if it’s the last open fltk window.

**Reply by Lydanne** [reply]
In this way, show is equivalent to creating a window, right?

**Reply by MoAlyousef** [reply]
Yes
